pom.xml

```diff
@@ -129,10 +129,10 @@
   </ciManagement>
 
   <properties>
-    <license.title>Sonar C++ Plugin (Community) </license.title>
+    <license.title>Sonar C++ Plugin (Community)</license.title>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-    <sonar.version>4.5.2</sonar.version>
-    <sslr.version>1.20</sslr.version>
+    <sonar.version>5.2</sonar.version>
```

Comment:
Looking to other plugins they seems to be alll still 4.5.2 compatible **and** working with SQ 5.2?
So this could still be 4.5.2?